HACKER/HACKER/Sources/ViewControllers/MainViewController/MainViewController.swift

```diff
@@ -0,0 +1,215 @@
+//
+//  MainViewController.swift
+//  HACKER
+//
+//  Created by 김지수 on 2022/03/18.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+// MARK: - MainViewController
+class MainViewController: UIViewController {
+  
+  // MARK: - Components
+  let nicknameLabel = UILabel()
+  let alarmButton = UIButton()
+  let settingsButton = UIButton()
+  let userCharacterImage = UIImageView()
+  let userhairImage = UIImageView()
+  let todayCommitLabel = UILabel()
+  let todayCommitNumLabel = UILabel()
+  let refreshButton = UIButton()
+  let progressBackgroundView = UIView()
+  let progressFrontView = UIView()
+  let attackContainerView = UIView()
+  let attackTicket = UIImageView()
+  let attackNum = UILabel()
+  
+  // MARK: - LifeCycle
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.view.backgroundColor = .hackerWhite
+    layout()
+    
```

Comment:
네비게이션바가 아직 살아있어서 safeLayoutGuides가 의도한대로 안 나오는 바람에 거기 대응되는 애들의 레이아웃이  11 Pro에서 조금 깨지는데 요거 추가해주면 될듯~!
<img width="684" alt="Screen Shot 2022-03-23 at 4 50 41 PM" src="https://user-images.githubusercontent.com/71601985/159650377-83847473-6ae1-4fef-8fda-d3b637318f04.png">

```suggestion
    self.navigationController?.navigationBar.isHidden = true
```